conftest.py

```diff
@@ -1,35 +1,10 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 # -*- coding: utf-8 -*-
 import os
-from collections import OrderedDict
 
 import pytest
 from _pytest.doctest import DoctestItem
 
-from astropy.tests.helper import enable_deprecations_as_exceptions
-from astropy.tests.plugins import display
-
-
-display.PYTEST_HEADER_MODULES = OrderedDict([
-                                    ('asdf', 'asdf'),
-                                    ('numpy', 'numpy'),
-                                    ('jsonschema', 'jsonschema'),
-                                    ('pyyaml', 'yaml'),
-                                    ('astropy', 'astropy')])
-
-try:
-    import gwcs
-    display.PYTEST_HEADER_MODULES['gwcs'] = 'gwcs'
-except ImportError:
-    pass
-
-
-pytest_plugins = [
-    'astropy.tests.plugins.display'
-]
-
-enable_deprecations_as_exceptions()
```

Comment:
Is this defined somewhere else or is it a change in testing?

Reply:
This removes this functionality and dependency on `astropy.tests.helpers` and `astropy.tests.plugins.display`.

That said `enable_deprecations_as_exceptions()` doesn't seem to work.  I.e. we are seeing deprecations yet the tests are passing.  So I'm not sure how useful this is.  The better way to do this is
```
pytest -W error::DeprecationWarning
```
which correctly catches deprecations exceptions.